src/libs/OmnibugSettings.js

```diff
@@ -49,6 +49,9 @@ class OmnibugSettings
             // keys to highlight
             highlightKeys  : ["pageName", "ch", "events", "products"],
 
+            // JSON defining the params to rename
+            renameParameters: "{ \"eVar1\": \"Page Title\" }",
```

Comment:
We'll want to have this empty as default, otherwise I promise you we're going to have a ton of "why does my eVar1 say page title" questions ha.

Reply:
ha good catch! will clean up